packages/generator/templates/app/test/setup.ts

```diff
@@ -2,4 +2,4 @@
 // allows you to do things like:
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
-import '@testing-library/jest-dom/extend-expect';
+import "@testing-library/jest-dom/extend-expect"
```

Comment:
**question**: Do we have a prettier config for the template files? It's funny to see a lot of changes in the style on this PR. I am not sure what should be the one to follow.

Reply:
It should be using the monorepo config. Not sure how the wrong format got in originally.